pkg/controller/ibmblockcsi/ibmblockcsi_controller.go

```diff
@@ -56,7 +56,12 @@ import (
 )
 
 // ReconcileTime is the delay between reconciliations
-const ReconcileTime = 30 * time.Second
+const (
+	ReconcileTime = 30 * time.Second
+)
```

Comment:
no need for round brackets (parenthesis)

Reply:
done